.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+Please describe the changes in your pull request in few words here.
+
+### Changes
+
+List the changes done to fix a bug or introducing a new feature.
+
+## How to test
+
+Describe the steps required to test the changes proposed in the pull request.
```

Comment:
not sure about this.

preferable a test added which covers the changes, but not always all can do that.  We mostly accept PR only related to an issue with a description